javascript/javascript2/week3/FuncAsVar.js

```diff
@@ -0,0 +1,34 @@
+//2.1
+const myFunction = [
+  (String = function () {
+    console.log('String')
+  }),
+  (bulean = function () {
+    console.log(true)
+  }),
+  (number = function () {
+    console.log(8)
+  }),
+]
+
+myFunction.forEach((i) => i())
+//2.2
+const objFun = {
+  String: function () {
+    console.log('String')
+  },
+  bulean: function () {
+    console.log(true)
+  },
+  number: () => {
+    console.log(4)
+  },
+}
```

Comment:
Here you should've created two functions using const and a normal function. And call them :thinking: